handlers/main.yml

```diff
@@ -4,11 +4,11 @@ handlers:
   command: /usr/bin/systemd-tmpfiles --prefix=/sys --create
 
 - name: Restart sshd
-  service:
+  systemd:
     name: sshd
     state: restarted
 
 - name: Restart fail2ban
-  service:
+  systemd:
```

Comment:
Using the service module is generic and it will pick the right init system. In Fedora's case that will be systemd.

Reply:
https://github.com/ansible/ansible/issues/43138 explains why i had to make the changes but we are back to using service